docs/docs/shapes/fitbox.md

```diff
@@ -0,0 +1,46 @@
+---
+id: fitbox
+title: FitBox
+sidebar_label: FitBox
+slug: /shapes/fibox
+---
+
+The `FitBox` components allows you to automatically scales drawings to you Canvas.
+
+| Name | Type     |  Description                                       |
+|:-----|:---------|:---------------------------------------------------|
+| src  | `SKRect` | Bounding rectangle of the drawings before scaling  |
```

Comment:
**Typo:**
| src  | `SKRect` | Bounding rectangle of the drawing before scaling  |

Reply:
done 🙌🏼